### PR TITLE
Tweak comment to fix html generation.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

**Comment by geeknoid** [comment]
@ostromart This also needs to be moved to the 1.3 branch in order to fix the docs.

**Review by howardjohn — APPROVED** [review]

**Review by sdake — APPROVED** [review]

**Comment by geeknoid** [comment]
@sdake Looks like this repo is currently broken. This we force-merge some changes?

**Comment by howardjohn** [comment]
postsubmit passed 19 minutes ago.. not sure why this fails

**Comment by sdake** [comment]
master works - this pr would break master - unclear why: https://github.com/istio/operator/pull/336

File: pkg/apis/istio/v1alpha2/istiocontrolplane_types.pb.go [hunk 1]
```diff
@@ -191,8 +190,6 @@ func (InstallStatus_Status) EnumDescriptor() ([]byte, []int) {
 
 // IstioControlPlane is a CustomResourceDefinition (CRD) describing an Istio control plane.
 type IstioControlPlane struct {
-	v11.ObjectMeta `json:"metadata,omitempty"`
```
Comment (sdake) [inline pkg/apis/istio/v1alpha2/istiocontrolplane_types.pb.go]: ObjectMeta and TypeMeta are required.  I am not sure why these are omiited in your PR.

Comment (geeknoid) [reply pkg/apis/istio/v1alpha2/istiocontrolplane_types.pb.go]: Oh, I missed that these were removed.

Arg, there's something funny happening with the code gen in this repo :-(

**Pull request merged**